src/gluonts/ext/statsforecast.py

```diff
@@ -198,16 +223,7 @@ class CrostonSBAPredictor(StatsForecastPredictor):
     ModelType = CrostonSBA
 
 
-class ETSPredictor(StatsForecastPredictor):
-    """
-    A predictor wrapping the ``ETS`` model from `statsforecast`_.
-
-    See :class:`StatsForecastPredictor` for the list of arguments.
-
-    .. _statsforecast: https://github.com/Nixtla/statsforecast
-    """
-
-    ModelType = ETS
+ETSPredictor = AutoETSPredictor  # NOTE: for backward-compatibility
```

Comment:
Not sure if it is better to just do a breaking change

Reply:
yeah we could also do that